js/default-table-data/layouts-table-config.js

```diff
@@ -1,3 +1,5 @@
+moment.locale('en');
```

Comment:
I wouldn't fix it this way. If the moment locale is globally changed to another language e.g. `ar` then you've essentially undone the setting.

Aside from changing the global moment locale, moment objects can be individually changed to have different locales, e.g. `moment().locale('ar')` and `moment().locale('it')` are 2 moment objects with different locales (without changing the global locale setting.

You should find the default data used in this file (I think it applies to Agenda and News feed, but please verify all types) and use something like this: `moment().add(1, 'month').locale('en').format('YYYY-MM-DD')`.